package.json

```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@hookform/resolvers": "^3.1.1",
-    "@petergok/tempo-devtools": "^2.0.33",
+    "tempo-devtools": "^2.0.36",
```

Comment:
You need to also switch all the places in the codebase where the tempo-devtools are used